src/SfdxProjectBuilder.groovy

```diff
@@ -90,6 +90,31 @@ class SfdxProjectBuilder implements Serializable {
   // the parsed contents of the SFDX project's configuration
   private def sfdxPackage
 
+  // dynamic property scaffold
+  private String packagingAlias() {
```

Comment:
@TheRickParker, I am not seeing this method used anywhere.  Am I missing something?